posthog/settings/dynamic_settings.py

```diff
@@ -128,6 +133,7 @@
     "ASYNC_MIGRATIONS_ROLLBACK_TIMEOUT",
     "ASYNC_MIGRATIONS_DISABLE_AUTO_ROLLBACK",
     "ASYNC_MIGRATIONS_AUTO_CONTINUE",
+    "ASYNC_MIGRATIONS_BLOCK_UPGRADE",
```

Comment:
```suggestion
    "ASYNC_MIGRATIONS_RUNNING_BLOCK_UPGRADE",
```

(Since there are other ways async migrations might block upgrades)